internal/app/secretless/handlers/mysql/connection_details_test.go

```diff
@@ -0,0 +1,83 @@
+package mysql
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConnectionDetailsCorrectlyParsesConnectionDetails(t *testing.T) {
```

Comment:
Since this is the test file for connection details (ie, we're in`connection_details_test`), we can dispense with `ConnectionDetails` in any of the names.  After reading through all the tests, it seems what we're actually testing here is that the basic, expected information is parsed correctly.

Therefore, suggested name: `TestExpectedFields`.

Additionally, all the tests here are testing different aspects of `NewConnectionDetails`.  We might consider:

```go
func TestNewConnectionDetails(t *testing.T) {
    t.Run("Test Expected Fields", func(t *testing.T) { ... })
    t.Run("Test Default Port", func(t *testing.T) { ... })
    // etc
}
```
though that's probably a premature optimization at this point, and improving the current names and relying on the file context is enough.